archetypes/helidon/src/main/archetype/se/common/files/src/test/java/__pkg__/MainIT.java.mustache

```diff
@@ -1,11 +1,11 @@
 package {{package}};
 
-import io.helidon.webclient.http1.Http1Client;
-import io.helidon.webserver.testing.junit5.ServerTest;
+import io.helidon.webserver.testing.junit5.DirectClient;
+import io.helidon.webserver.testing.junit5.RoutingTest;
 
-@ServerTest
+@RoutingTest
```

Comment:
Shouldn't the IT test use `@ServerTest` and the unit test use `@RoutingTest` ?

Reply:
Yes you are right, it was a workaround. It should not have been in the PR.